.github/workflows/codeql-analysis.yml

```diff
@@ -0,0 +1,63 @@
+# For most projects, this workflow file will not need changing; you simply need
+# to commit it to your repository.
+#
+# You may wish to alter this file to override the set of languages analyzed,
+# or to provide custom queries or build logic.
+name: "Static Application security Testing (CodeQL)"
+
+on:
+  push:
+    branches:
+    - '*'
+  pull_request:
+    # The branches below must be a subset of the branches above
+    branches:
+    - master
+  schedule:
+    - cron: '0 12 * * 6'
+
+jobs:
+  analyze:
+    name: Analyze
+    runs-on: ubuntu-latest
+
+    strategy:
+      fail-fast: false
+      matrix:
+        language: [ 'javascript', 'python' ]
```

Comment:
Is supporting python required here for the API documentation ?

Reply:
I thought so, although I didn't think of it when I enabled SAST on solid-client. If it doesn't help there's no harm in including it either, I think.